angr_platforms/risc_v/arch_riscv.py

```diff
@@ -1,60 +1,171 @@
+try:
+    import capstone as _capstone
+except ImportError:
+    _capstone = None
+
+from .arch import Arch, register_arch, Endness, Register
+from .archerror import ArchError
+from .tls import TLSArchInfo
+
 from archinfo.arch import register_arch, Arch, Register
 from archinfo.tls import TLSArchInfo
```

Comment:
Could you remove one of these sets of lines? The imports are duplicated here

Reply:
actually you have to remove the green lines because we're not in archinfo anymore so you can't do relative imports